AmplifyPlugins/Logging/Tests/AWSCloudWatchLoggingPluginTests/LogActorTests.swift

```diff
@@ -109,4 +109,22 @@ final class LogActorTests: XCTestCase {
         XCTAssertTrue(contents.isEmpty)
     }
     
+    
+    /// Given: a LogActor
+    /// When: get all logs is called
+    /// Then: all logs are returned
+    func testLogActorReturnsLogList() async throws {
+        var logs = try await systemUnderTest.getLogs()
+        XCTAssertEqual(logs.count, 1)
+        let size = try LogEntry.minimumSizeForLogEntry(level: .error)
+        let numberOfEntries = (fileSizeLimitInBytes/size) + 1
```

Comment:
does this mean `numberOfEntries` is equals to 1?

Reply:
I've updated the test documentation with the precondition.